packages/elements-core/src/components/Docs/HttpService/HttpService.tsx

```diff
@@ -15,45 +17,76 @@ import { ServerInfo } from './ServerInfo';
 
 export type HttpServiceProps = DocsComponentProps<Partial<IHttpService>>;
 
-const HttpServiceComponent = React.memo<HttpServiceProps>(({ data, location = {}, layoutOptions, exportProps }) => {
-  const { search, pathname } = location;
-  const mocking = React.useContext(MockingContext);
-  const query = new URLSearchParams(search);
-  return (
-    <Box mb={10}>
-      {data.name && !layoutOptions?.noHeading && (
-        <Flex justifyContent="between" alignItems="center">
-          <Heading size={1} mb={4} fontWeight="semibold">
-            {data.name}
-          </Heading>
-          {exportProps && !layoutOptions?.hideExport && <ExportButton {...exportProps} />}
-        </Flex>
-      )}
-      {data.version && (
-        <Box mb={5}>
-          <VersionBadge value={data.version} />
-        </Box>
-      )}
-      {pathname && layoutOptions?.showPoweredByLink && (
-        <PoweredByLink source={data.name ?? 'no-title'} pathname={pathname} packageType="elements" layout="stacked" />
-      )}
-      <VStack spacing={6}>
-        <ServerInfo servers={data.servers ?? []} mockUrl={mocking.mockUrl} />
-        <Box>
-          {data.securitySchemes?.length && (
-            <SecuritySchemes schemes={data.securitySchemes} defaultScheme={query.get('security') || undefined} />
-          )}
-        </Box>
-        <Box>
-          {(data.contact?.email || data.license || data.termsOfService) && (
-            <AdditionalInfo contact={data.contact} license={data.license} termsOfService={data.termsOfService} />
-          )}
-        </Box>
-      </VStack>
-      {data.description && <MarkdownViewer className="sl-my-5" markdown={data.description} />}
-    </Box>
-  );
-});
+const HttpServiceComponent = React.memo<HttpServiceProps>(
```

Comment:
Prettier decided to indent this more which caused a large diff here - in reality very little has changed.